msteams-platform/toolkit/add-capability.md

```diff
@@ -68,10 +68,11 @@ Apart from the capabilities your Teams app already have, you can choose to add d
 |Tabs with SPFx|None|
 |Tabs with Azure|Bots and messaging extensions|
 |Bots|Tabs|
-|Messaging extensions|Tabs|
-|Tabs and bots|None|
-|Tabs and messaging extensions|None|
-|Tabs, bots, and messaging extensions|None|
+|Messaging extensions|Tab and Bot|
```

Comment:
How about using singular terms for `bot` and `messaging extension`, otherwise people may be confused if they can add multiple bots and messaging extensions.